Description: Fix FTBFS for gcc-4.6
 Add -Wno-error=unused-but-set-variable -Wno-error=unused-but-set-parameter
 -Wno-error=deprecated-declarations for CFLAGS.
Author: Ying-Chun Liu (PaulLiu) <paulliu@debian.org>
Bug-Debian: http://bugs.debian.org/625322
Last-Update: 2012-06-12
Index: clutter-gesture-0.0.2.1/configure.ac
===================================================================
--- clutter-gesture-0.0.2.1.orig/configure.ac	2010-01-06 12:11:44.000000000 +0800
+++ clutter-gesture-0.0.2.1/configure.ac	2012-06-12 15:47:42.238404969 +0800
@@ -50,7 +50,7 @@
 			clutter-1.0 >= 1.0.0
 			gobject-2.0
 			glib-2.0)
-CLUTTERGESTURE_CFLAGS="$CLUTTERGESTURE_CFLAGS -Wall -Werror -fPIC"
+CLUTTERGESTURE_CFLAGS="$CLUTTERGESTURE_CFLAGS -Wall -Werror -Wno-error=unused-but-set-variable -Wno-error=unused-but-set-parameter -Wno-error=deprecated-declarations -fPIC"
 AC_SUBST(CLUTTERGESTURE_CFLAGS)
 AC_SUBST(CLUTTERGESTURE_LIBS)
 
